.github/workflows/update-sources.yml

```diff
@@ -23,6 +28,9 @@ jobs:
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: master
```

Comment:
if this is just a speedup of clone execution, please move it to separate PR to merge sooner.

Reply:
It is not a speedup of execution. This one in particular can be removed, but the rest of
```yml
        with:
          fetch-depth: 0
          ref: master
```
that I added are required when we have a job that is dependent on a previous job that commits and pushes to repo,

There is an issue for this at `actions/checkout`:https://github.com/actions/checkout/issues/439#issue-804387669

I use the solution proposed in the comments: https://github.com/actions/checkout/issues/439#issuecomment-965968956